apps/vc-api/test/vc-api/workflows/resident-card/resident-card.e2e-suite.ts

```diff
@@ -174,7 +174,7 @@ export const residentCardWorkflowSuite = () => {
     const vp = await walletClient.provePresentation({ presentation, options: presentationOptions });
 
     // Holder submits presentation
-    await walletClient.continueWorkflowExchange(presentationExchangeContinuationEndpoint, vp, 'empty');
+    await walletClient.continueWorkflowExchange(presentationExchangeContinuationEndpoint, vp, 'redirect');
```

Comment:
Can use the existing `redirectUrl` type
```suggestion
    await walletClient.continueWorkflowExchange(presentationExchangeContinuationEndpoint, vp, 'redirectUrl');
```